bsip-0022.md

```diff
@@ -9,113 +9,439 @@
+
+
+### Changing the Decay Steps of a Referendum Category or Proxy Delegation
+
+Changing only the quantity of decay steps can potentially change the voting power of accounts _whose voting age fall within the decay period_.  The change in voting power for these accounts is a function of both the change in the quantity of decay steps, and the account's voting age.
+
+
+## <div id="initial-voting-power" /> Initial Voting Power
+
+Vote decay will have no effect on any account prior to the activation of this proposal with a hardfork date ("activation date").  After this proposal is activated, the **effective** [latest vote date](https://github.com/bitshares/bitshares-core/blob/bf7ff54d9a17aa43f4663521e371b8c0ddfc2284/libraries/chain/include/graphene/account_object.hpp#L73) for the purpose of vote decay calculations, shall be considered to be the activation date.  The activation date will effectively become the [start of the full-voting period](#vote-decay-function) for every account.  All votes that exist at the time of the activation date will initially have maximum voting power across every referendum category and proxy delegation.
+
+Any subsequent voting by an account will set a new latest vote date for an account that will reset the [vote decay clock](#separate-decay-rates) for that account.
+
+
+## Comparisons
+
+|Element|Proposal|[Peerplays GPOS](#peerplays-vote-decay)|[EOS](#eos-vote-decay)|
+|-|-|-|-|
+|Decay Rate|Full-power during initial period followed by stepped decrease during decay period until reaching zero power|Full-power during initial sub-period followed by stepped decrease until reaching zero power|Full-power during initial period followed by stepped increase of newer votes by other accounts|
+|Effect of decay rate on referendum issues|Separate decay rates for every _decayable_ referendum category and proxy delegation|Single decay rate shared by every referendum category|Single decay rate shared by every referendum category|
+|Effect of decay rate on other issues|None|Decays the share of blockchain profits that are received by an account|Decays the share of payment for standby block producers|
+
+## <div id="software-specs"/> Software Specifications
+
+The following specifications are proposed for implementation.
+
+**Note 1:** Calculation of durations (e.g. days) are independent of the local calendar and timezone (e.g. Daylight Savings Time is ignored)
+
+**Note 2:** "Implementation hints" are not to be considered part of the formal specification, but merely as a *possible* implementation.
+
+### Vote Decay parameters
+
+The following vote decay parameters shall be selectable by the BitShares Committee.
+
+```
+struct category {
+   bool active;
+   unsigned_int full_power_duration; // days
```

Comment:
Note: "days" means periods of 86400 seconds. Local calendars are ignored, as are DST and other irregularities.